osi_analytic_segments_defaults/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 2021 - TODAY, Open Source Integrators
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "OSI Analytic Segments Defaults",
+    "version": "14.0.1.0.0",
+    "license": "LGPL-3",
+    "summary": "Additional analytic segments for Analytic Defaults",
+    "author": "Open Source Integrators",
+    "maintainer": "Open Source Integrators",
+    "website": "http://www.opensourceintegrators.com",
+    "category": "Analytic Accounting",
+    "depends": [
+        "purchase",
+        "account",
+        "osi_analytic_segments",
```

Comment:
This module also depends on sale (I received an error that sale.order.line doesn't exist in registry when trying to install it.)
Please add "sale" to depends.

Reply:
Added